appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -84,7 +84,7 @@ internal sealed class ExperienceLifecycleEvent(
         get() = hashMapOf<String, Any?>(
             "experienceId" to experience.id.appcuesFormatted(),
             "experienceName" to experience.name,
-            "experienceType" to experience.type,
+            "experienceType" to experience.renderContext.toString(),
```

Comment:
this is an important callout since we have a override fun toString() that returns the experienceType.